src/sql-parser/src/parser.rs

```diff
@@ -2622,13 +2622,12 @@ impl<'a> Parser<'a> {
     fn parse_create_subsource_option_name(
```

Comment:
the docstring seems incorrect for this method

Reply:
Fixed!